serverless.env.yaml

```diff
@@ -0,0 +1,10 @@
+vars: null
```

Comment:
removed the file and verified with a fresh deploy, updated the PR with the removal